app/models/low_query_ctr_watcher.rb

```diff
@@ -10,24 +12,31 @@ def humanized_alert_threshold
     "#{low_ctr_threshold}% CTR on #{number_with_delimiter search_click_total} Queries & Clicks"
   end
 
+  def label
+    'Low Query Click-Through Rate (CTR)'
+  end
+
+  private
+
   def input(json)
-    options = { affiliate_name: affiliate.name, time_window: time_window, min_doc_count: search_click_total.to_i,
+    options = { affiliate_name: affiliate.name,
+                time_window: time_window,
+                min_doc_count: search_click_total.to_i,
                 query_blocklist: query_blocklist }
     low_query_ctr_query_body = WatcherLowCtrQuery.new(options).body
-    input_search_request(json, search_type: :count, types: %w(search click),
-                         indices: watcher_indexes_from_window_size(time_window), body: JSON.parse(low_query_ctr_query_body))
+    input_search_request(json,
+                         types: %w[search click],
+                         indices: watcher_indexes_from_window_size(time_window),
+                         body: JSON.parse(low_query_ctr_query_body).merge(size: 0))
   end
 
   def condition_script
-    "ctx.payload.aggregations && ctx.payload.aggregations.agg.buckets.any({ it.ctr.value < #{low_ctr_threshold}})"
+    "ctx.payload.aggregations.agg.buckets.any(it -> it.ctr.value < #{low_ctr_threshold})"
   end
 
   def transform_script
-    "ctx.payload.aggregations.agg.buckets.findAll({ it.ctr.value < #{low_ctr_threshold}}).collect({ it.key }).join('\",\"')"
+    # rubocop:disable LineLength
```

Comment:
I'm allowing this line to run long as it makes it easier to work with for debugging and comparing in Kibana.